event/eth_sign_redeem.go

```diff
@@ -87,20 +87,20 @@ func (j *JobETHSignRedeem) DoMyJob(ctx interface{}) {
 	tx, err := cd.DecodeTransaction(rawTx)
 	if err != nil {
 		ethCtx.Logger.Error("Error Decoding Bytes from RaxTX :", j.GetJobID(), err)
-		return
+		//return
```

Comment:
This has to be implemented thoroughly.. I m not sure if its good idea to remove the return here. Which means, even though the decode fails, it will go to line 93 to get the transaction message. Is that a good idea?

If the oETH is already burnt in these cases, then we should refund in all these cases, instead of just returning here...

You also need to consider the situation of lock here...Not just redeem